pkgs/development/python-modules/furl/default.nix

```diff
@@ -35,6 +36,11 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  disabledTests = lib.optionals (pythonAtLeast "3.12") [
```

Comment:
This applies to 3.10.15, 3.9.20, 3.8.20 too, https://pythoninsider.blogspot.com/2024/09/python-3130rc2-3126-31110-31015-3920.html

> [gh-67693](https://github.com/python/cpython/issues/67693): Fix [urllib.parse.urlunparse()](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlunparse) and [urllib.parse.urlunsplit()](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlunsplit) for URIs with path starting with multiple slashes and no authority. Fixes CVE-2015-2104.

Reply:
Feel free to send a PR.

Reply:
Pushed as 222d7063940ddf7d60b86ad81d54663b4357254a

Reply:
Oh, I didn't realize that you had a commit for this already in the other PR.

Reply:
It's fine, I don't mind.